apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -635,13 +634,6 @@ export function renderError({
   iconColor?: string;
   device?: Device;
   hasExportLogButton?: boolean;
-  /*
-   * Used when rendering a Bluetooth disabled error
-   *
-   * If "drawer", the component will be rendered as a content to be rendered in a drawer.
-   * If "view", the component will be rendered as a view. Defaults to "view".
-   */
-  renderedInType?: "drawer" | "view";
```

Comment:
this has been removed, it's used only in `BluetoothDisabled` and it was replaced by a more practical `IsInDrawerContext`